src/reader.rs

```diff
@@ -12,15 +12,15 @@ use crate::raw_reader::{RawReader, RawStreamItem};
 use crate::raw_symbol_token::RawSymbolToken;
 use crate::result::{decoding_error, decoding_error_raw, IonResult};
 use crate::stream_reader::IonReader;
-use crate::symbol::Symbol;
 use crate::symbol_table::SymbolTable;
-use crate::types::decimal::Decimal;
-use crate::types::integer::Int;
-use crate::types::timestamp::Timestamp;
+use crate::types::Decimal;
+use crate::types::Int;
+use crate::types::Symbol;
+use crate::types::Timestamp;
```

Comment:
As I look through a bunch of these, how much should be be consistent with `types::{...}` style imports?